lib/themes/dosomething/paraneue_dosomething/js/components/FeatureCard.js

```diff
@@ -189,7 +195,7 @@ class FeatureCard extends React.Component {
               <p className="tagline">{campaign.tagline}</p>
               <ul className="form-actions">
                 {(campaign.signedUp && !this.state.isSignedUp) ? <h1>You're signed up already!</h1> : <li><button className={`${this.state.isSignedUp ? 'tada' : ''} button`} onClick={()=>this.signup()}>Sign Up</button></li>}
-                <li><a className="button -tertiary" onClick={()=>this.viewAnother()}>Show me another</a></li>
+                <li><a className="button -tertiary" onClick={()=>this.viewAnother(true)}>Show me another</a></li>
```

Comment:
I was thinking a possible alternative would be doing both the ga() call here and view more, but because its not rendered right in my editor or here i figured it would be really ugly to read